CHANGELOG.md

```diff
@@ -34,6 +34,8 @@
 
 ### Added
 
+- [#3379](https://github.com/ChainSafe/forest/pull/3379): Zero-copy DAG_CBOR
+  into Vec<Cid> parser.
```

Comment:
This is not user-visible addition. I don't think it should be in the CHANGELOG. If we want a CHANGELOG entry, it should be under `Changed`, and it should say the performance was improved when walking the state graph.

Reply:
Sounds good.